src/megameklab/com/printing/PrintMech.java

```diff
@@ -533,51 +533,6 @@ protected void drawFluffImage() {
         }
     }
 
-    private void drawHeatSinkPips(SVGRectElement svgRect) {
```

Comment:
How's the heat sink pips look now for mechs?

Reply:
It's in a series of columns in the heat block. This doesn't change the appearance. It just moves it into the parent class so it can be available to both PrintMech and PrintAero.